modules/Virtualizer_module/CMakeLists.txt

```diff
@@ -44,8 +38,7 @@ yarp_add_idl(${EXE_TARGET_NAME}_THRIFT_SRC ${${EXE_TARGET_NAME}_THRIFT_HDR})
 
 # add include directories to the build.
 include_directories(
-  ${CMAKE_CURRENT_SOURCE_DIR}/include
-  ${CMAKE_CURRENT_SOURCE_DIR}/CybSDK/include)
+  ${CMAKE_CURRENT_SOURCE_DIR}/include )

```

Comment:
You can substitute this with [``target_include_directories``](https://cmake.org/cmake/help/v3.3/command/target_include_directories.html). The difference is that you are more specific in which target that folder should be used. Even if for executables this does not change anything, for libraries it is more important as it determines where headers can be found.

Reply:
done!